imagesgallery/imagesgallery.py

```diff
@@ -74,22 +89,57 @@ def student_view(self, context=None):
         if context:
             pass  # TO-DO: do something based on the context.
 
-        # Remove the bundle default generated by webpack
-        script_to_remove = r'<script defer="defer" src="bundle.js"></script>'
+        xblock_id = self.block_id
+        # Split the string by the @ symbol and get the last part
+        # XBlock IDs typically follow this format: block-v1:{course_name}+type@{xblockname}+block@{id_xblock}
+        # This code extracts the last part of the string, which is {id_xblock}
+        xblock_id_splitted = xblock_id.split('@')[-1]
```

Comment:
You can obtain the `block_id` by accessing `self.scope_ids.usage_id.block_id`, and then we can remove the `split`

Reply:
Thanks! this is better